docker-compose.yml

```diff
@@ -34,13 +35,16 @@ services:
       - default
     ports:
       - "5000"
+      # 0.0.0.0:5000:5000 # To access directly outside of amazeeio haproxy wiring
     image: *project
     <<: *default-volumes
     environment:
       <<: *default-environment
       AMAZEEIO_HTTP_PORT: 5000
       LAGOON_LOCALDEV_URL: "http://${PROJECT}.docker.amazee.io"
       AMAZEEIO_URL: "${PROJECT}.docker.amazee.io"
+    #command: # Used to start container but not serve so you can reboot without issues
```

Comment:
@ThrawnCA , how could we make this be uncommented for 'local' run where we wish to be able to 'copy' in new code without doing a 'full' build again.

Reply:
I've never succeeded in doing that before, so I'm not sure, sorry, but if you do figure it out then I'll be quite interested.

Reply:
would have to be another sed pattern like #%# that is used for alt pattern.